examples/tutorials/writing-and-serving-policy/controlling-frequency.markdown

```diff
@@ -62,3 +104,27 @@ atomic promise checks on the same objects (packages, users, files,
 etc.). Several different `cf-agent` instances can run concurrently.
 The locks ensure that promises will not be verified by two cf-agents
 at the same time or too soon after a verification.
+
+For example, here the `sshd` package promises to be at the latest version. It
+has the `if_elapsed_day` action body attached which sets `ifelapsed` to `1440`
+causing the promise lock to persist for a day effectively restricting the
+promise to run just once a day.
+
+```cf3
+bundle agent __main__
+{
+  packages:
+    "sshd"
+      version => "latest",
+      action => if_elapsed_day,
+      comment => "Make sure sshd is at the latest version, but just once a day.";
+}
+```
+
+Note: Promise locks are ignored when CFEngine is run with the `--no-lock` or
+`-K` option, e.g. a common **manual** execution of the agent, `cf-agent -KI`
+would not respect promises that are locked from a recent execution. Furthermore,
+locks are purged in order to maintain the integrity and health of the underlying
```

Comment:
maybe clarify here that the locks are purged when the agent is run with `--no-lock`?

Reply:
Are they *purged* when run with `-K`?

Reply:
`--no-lock/-K` means that locks are ignored, but they are not purged. Or, more precisely, they are purge just like in any other agent run.

Reply:
Yeah, originally I said ignored, ill change it back.

Reply:
They are separate statements, I can bullet point them out and I can add a bit of text with the `Furthermore ...` indicating purging based on db utilization.